analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp6.cs

```diff
@@ -170,12 +170,12 @@ public void Method2(ReproForIssue2338 obj)
             switch (obj?.Color)
             {
                 case ConsoleColor.Red:
-                    Console.ForegroundColor = obj.Color;  // Noncompliant FP, was compliant in the old engine. Should be fixed by MMF-2401
-//                                            ^^^    {{'obj' is null on at least one execution path.}}
+                    Console.ForegroundColor = obj.Color;  // Non-compliant Suppressed #6117 FP, was compliant in the old engine. Should be fixed by MMF-2401
+                    //                        ~~~            {{'obj' is null on at least one execution path.}}
                     break;
                 default:
-                    Console.WriteLine($"Color {obj.Color} is not supported."); // Noncompliant
-//                                             ^^^    {{'obj' is null on at least one execution path.}}
+                    Console.WriteLine($"Color {obj.Color} is not supported."); // Non-compliant Suppressed #6117
+                    //                         ~~~                                {{'obj' is null on at least one execution path.}}
```

Comment:
I would remove this. The ~~~ makes sense, but it's too easy to miss when restoring the FP
```suggestion
```

Reply:
Same above